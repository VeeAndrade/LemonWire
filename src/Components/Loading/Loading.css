.lemon-img {
  height: 300px;
  width: 300px;
  margin-top: 40%;
}

.loading-text {
  position: relative;
  top: 67%;
  left: 28%;
  font-size: 200%;
  font-weight: bolder;
}
