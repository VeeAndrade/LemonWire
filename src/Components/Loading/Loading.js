import React from 'react';
import { Route, Redirect } from 'react-router-dom';
import lemon from '../../Assets/lemon.png';
import './Loading.css';

const Loading = () => {
  return (
    <section>
      <p className='loading-text'>loading...</p>
      <img src={lemon} alt='loading lemon' className='lemon-img'/>
    </section>
  )
}

export default Loading;
