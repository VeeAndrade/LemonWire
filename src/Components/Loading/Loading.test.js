import React from 'react';
import Loading from './Loading';
import { shallow } from 'enzyme';

describe('Loading', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })

  it ('should render a Loading component', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
