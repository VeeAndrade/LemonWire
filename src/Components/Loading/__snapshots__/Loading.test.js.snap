// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render a Loading component 1`] = `
<section>
  <p
    className="loading-text"
  >
    loading...
  </p>
  <img
    alt="loading lemon"
    className="lemon-img"
    src="lemon.png"
  />
</section>
`;
